src/Jank/TermManagement/Term.tsx

```diff
@@ -1,15 +1,31 @@
-import React from 'react';
-import { DeleteButton, TermText } from './ui';
+import React, { useCallback } from 'react';
+
+import useErrorHandler from '../../hooks/useErrorHandler';
+import { remove } from '../../services/api.service';
+import { Action } from './types';
+import { DeleteButton, TermText, ErrorText } from './ui';
+import { deleteTerm } from './actions';
 
 type TermProps = {
-  value: string
+  id: string,
+  value: string,
+  dispatch: (action: Action) => void
 }
 
-const Term = ({ value } : TermProps) => (
-  <li>
-    <TermText>{value}</TermText>
-    <DeleteButton type="button">X</DeleteButton>
-  </li>
-);
+const Term = ({ id, value, dispatch }: TermProps) => {
+  const callback = useCallback(async () => {
+    await remove(`/jank/terms/${id}`);
+    dispatch(deleteTerm(id));
+  }, [dispatch, id]);
+  const [error, onClick] = useErrorHandler(callback);
```

Comment:
Brief explanation of what's going on here:
* We use a custom `useErrorHandler()` hook which I added in this PR - this is so we don't need to add error handling everywhere and can just re-use this hook. We can add more elaborate error handling there later on.
* The hook uses a callback for which the errors are handled. The callback does two things:
1. Call the endpoint to actually delete the term.
2. Dispatch an action to update our local state, causing the UI to re-render so the term actually disappears.

The dispatch is taken from the `useReducer()` hook below. I'll add some more details there.